examples/2016-10-31/policy_templates/all_policy_templates.yaml

```diff
@@ -101,3 +101,5 @@ Resources:
 
         - CodeCommitReadPolicy:
             RepositoryName: name
+
+        - EventBridgePutEventsPolicy: {}
```

Comment:
As discussed in the corresponding issue, let's add an EventBus parameter to this policy template so the permissions can be scoped to a specific event bus. See other policy templates that take a parameter as an example.